digitalsadhu/otf-battlemaps
# Added dark mode, more readable text, gridline transparency

### Justification
Options that were talked about on the discord.

### Test Procedure - Automated
Ensure unit tests pass

### Test Procedure - Manual
1. Provide a url including the `opt=d` option
    Ensure that the map appears with a dark background and light grid text
2. Provide a url including the `opt=gh` option
    Ensure that the map appears with gridlines that are slightly transparent. Test will probably be more obvious if a background is given
3. Provide a url including the `opt=g0` option
    Ensure that the map appears with no gridlines, though the border is still defined
4. Smoke test these options when provided together. 
    Note that if both `gh` and `g0` are provided, the `gh` option is prioritized. imo this is fine as providing both isn't a valid use case.
